executor/index_lookup_merge_join.go

```diff
@@ -423,8 +424,10 @@ func (imw *innerMergeWorker) handleTask(ctx context.Context, task *lookUpMergeJo
 		sort.Slice(task.outerOrderIdx, func(i, j int) bool {
 			idxI, idxJ := task.outerOrderIdx[i], task.outerOrderIdx[j]
 			rowI, rowJ := task.outerResult.GetRow(idxI), task.outerResult.GetRow(idxJ)
-			for id, joinKey := range imw.outerMergeCtx.joinKeys {
-				cmp, _, err := imw.outerMergeCtx.compareFuncs[id](imw.ctx, joinKey, joinKey, rowI, rowJ)
+			for idxOff := range imw.joinKeys {
```

Comment:
It makes me confusion here: Why we loop over the join keys but get index offset? What about just loop over the `idxOff2KeyOff` after making it an array?